scripts/make_submission_file.py

```diff
@@ -121,6 +140,9 @@ def main(args):
         action="store_true",
         help="Write IS2RE results from S2EF prediction files. Paths specified correspond to S2EF NPZ files.",
     )
+    parser.add_argument(
+        "--oc22", action="store_true", help="Write OC22 prediction files."
+    )
```

Comment:
Let's make this arg `split` or `dataset` and default to `oc20`, that way you can remove the duplication in lines 74-81.

Reply:
I added a `dataset` flag with OC20 as the default. Additionally, I removed the duplicated code in lines 74-81 and removed the `dataset` variable in favor of `args.dataset`.